src/YourEasyRent/Services/TgButtonCallback.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Microsoft.VisualBasic;
+using System.Numerics;
+using Telegram.Bot.Types;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace YourEasyRent.Services
+{
+    public class TgButtonCallback
+    {
+        private readonly Update _update;
+        public bool IsStart { get; set; }
+        public bool IsValidMessage {  get; set; }
+        public bool IsValueMenuMessage { get; set; } 
+        public bool IsBrandMenu {  get; set; }  
+        public bool IsCategoryMenu { get; set; }    
+        public bool IsValueProductButton {  get; set; }
+
+
+        public TgButtonCallback(Update update)
+        {
+            _update = update;
+        }
+
+        public void IsBotStart()
```

Comment:
change it to public bool IsBotStart() and return the result. Remove public properties

Reply:
if i make this method GetIsStart() "private" how I'll call it in another class?

Reply:
i use this property "IsStart" for TelegramCallbackHandler class  -   public async Task HandleUpdateAsync(TgButtonCallback tgButtonCallback) 
  {var userSearchState = new UserSearchState();
      if (tgButtonCallback.IsStart)...}